compiler/tests-compiler/gen-rules/gen.ml

```diff
@@ -60,6 +60,13 @@ let test_enabled_if = function
   | "gh1051" -> B64
   | _ -> Any
 
+let enabled_if cond =
+  match cond with
+  | Any -> "(and (<> %{profile} wasm) (<> %{profile} wasm-effects))"
+  | GE5 ->
+      "(and (<> %{profile} wasm) (<> %{profile} wasm-effects) (>= %{ocaml_version} 5))"
+  | B64 -> "(and (<> %{profile} wasm) (<> %{profile} wasm-effects) %{arch_sixtyfour})"
+
```

Comment:
Can't we do something for the entire directory instead of duplicating `(and (<> %{profile} wasm) (<> %{profile} wasm-effects))`

Reply:
I have simplified a bit the generated rules. But I don't see how to do something for the entire directory at once.

Maybe we could use environment variables to test whether we are generating JS or Wasm code, and for effects rather than matching on the profile's name?